### PR TITLE
Fix num of partitions is wrong

### PR DESCRIPTION
**Pull request by smellthemoon**
Signed-off-by: lixinguo [xinguo.li@zilliz.com](mailto:xinguo.li@zilliz.com)
/kind improvement
/kind bug

### CODE REVIEW

File: internal/rootcoord/meta_table.go [hunk 1]
```diff
@@ -547,9 +547,8 @@ func (mt *MetaTable) ChangePartitionState(ctx context.Context, collectionID Uniq
 
 			switch state {
```
Comment (bigsheeper) [inline internal/rootcoord/meta_table.go:548]: just remove swith case?

**Comment by smellthemoon** [comment]
rerun ut

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**